include/nncase/runtime/stackvm/opcode.h

```diff
@@ -161,7 +161,8 @@ enum class tensor_function_t
     TOPK = 0x0024,
     TRANSPOSE = 0x0025,
     TRILU = 0x0026,
-    UNARY = 0x0027,
+    TFLITE_DETECTION_POSTPROCESS = 0x0027,
```

Comment:
Leave `UNARY = 0x0027` or it will introduce a breaking change.

Reply:
Done